crates/net/src/types/bootnode.rs

```diff
@@ -0,0 +1,33 @@
+use ethrex_core::H512;
+use std::{net::SocketAddr, num::ParseIntError, str::FromStr};
+
+#[derive(Debug, Clone)]
+pub struct BootNode {
+    pub node_id: H512,
```

Comment:
This should be `H256` since the ID is a hash.

Reply:
According to the documentation it's a pubkey (probably secp256k1), of 64 bytes (512 bits)